pkg/crds/crds_test.go

```diff
@@ -21,43 +21,25 @@ import (
 	opv1 "github.com/tigera/operator/api/v1"
 )
 
+// The real test here is simply calling these functions will result in a panic if any of the CRDs cannot be parsed
+
 var _ = Describe("test crds pkg", func() {
-	Context("GetCalicoCRDSource", func() {
-		Measure("should quickly load calico source CRDs", func(b Benchmarker) {
-			runtime := b.Time("runtime", func() {
-				_ = getCalicoCRDSource()
-			})
-			Expect(runtime.Seconds()).Should(BeNumerically("<", 0.3), "loading calico CRDs should not take too long.")
-		}, 50)
+	It("can parse Calico CRDs", func() {
+		Expect(getCalicoCRDSource()).ToNot(BeEmpty())
```

Comment:
I think you can do this as well:

```go
Expect(getCalicoCRDSource).NotTo(Panic())`
```

If we want to test for panics - this should catch the panic rather than bringing the whole test suite